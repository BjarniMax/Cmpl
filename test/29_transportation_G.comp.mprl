<?xml version="1.0" encoding="UTF-8"?>
<mprl xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.coliop.org/schemas/Mprl.xsd">
	<general>
		<generalStatus>normal</generalStatus>
		<instanceName>29_transportation_G.cmpl</instanceName>
		<mplName>CMPL</mplName>
		<message>cmpl finished normal</message>
	</general>
</mprl>