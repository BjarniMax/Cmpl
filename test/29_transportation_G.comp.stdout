---------------------------------------------------------------------------------------------------------
Problem              29_transportation_G.cmpl
Nr. of variables     12
Nr. of constraints   7
Objective name       costs
Solver name          Gurobi
---------------------------------------------------------------------------------------------------------

Objective status     integer optimal solution
Objective value      39500 (min!)

Variables 
Name                 Type            Activity         Lower bound         Upper bound            Marginal
---------------------------------------------------------------------------------------------------------
x[1,1]                  I                3500                   0            Infinity                   -
x[1,2]                  I                1500                   0            Infinity                   -
x[1,3]                  I                   0                   0            Infinity                   -
x[1,4]                  I                   0                   0            Infinity                   -
x[2,1]                  I                   0                   0            Infinity                   -
x[2,2]                  I                2500                   0            Infinity                   -
x[2,3]                  I                2000                   0            Infinity                   -
x[2,4]                  I                1500                   0            Infinity                   -
x[3,1]                  I                2500                   0            Infinity                   -
x[3,2]                  I                   0                   0            Infinity                   -
x[3,3]                  I                   0                   0            Infinity                   -
x[3,4]                  I                   0                   0            Infinity                   -
---------------------------------------------------------------------------------------------------------

Constraints 
Name                 Type            Activity         Lower bound         Upper bound            Marginal
---------------------------------------------------------------------------------------------------------
supplies_1              L                5000           -Infinity                5000                   -
supplies_2              L                6000           -Infinity                6000                   -
supplies_3              L                2500           -Infinity                2500                   -
demands_1               E                6000                6000                6000                   -
demands_2               E                4000                4000                4000                   -
demands_3               E                2000                2000                2000                   -
demands_4               E                1500                1500                1500                   -
---------------------------------------------------------------------------------------------------------
